src/Illuminate/Database/Console/ShowModelCommand.php

```diff
@@ -106,10 +106,9 @@ public function handle()
      */
     protected function getPolicy($model)
     {
-        return collect(Gate::policies())
-            ->filter(fn ($policy, $modelClass) => $modelClass === get_class($model))
-            ->values()
-            ->first();
+        $policy = Gate::getPolicyFor($model::class);
+
+        return $policy ? $policy::class : '';
```

Comment:
The existing method returns `null` when no policy is found (despite the docblock saying otherwise). This doesn't affect the CLI output, but it does change the JSON output.

I'd suggest we do this instead (and update the doc block to be correct)

```suggestion
        return $policy ? $policy::class : null;
```